.cirrus.yml

```
@@ -21,6 +21,24 @@ task:
     - export PATH="/tmp/corral/bin:/tmp/ponyc/bin/:$PATH"
     - gmake test ssl=1.1.x
 
+task:
+  only_if: $CIRRUS_PR != ''
+
+  freebsd_instance:
+    image: freebsd-12-1-release-amd64
```

We should use a FreeBSD 12.2 image for this.

```suggestion
    image: freebsd-12-2-release-amd64
```

---

We are testing on 13 now so we should switch to it (see changes in ponyc repo and corral)